packages/better-php-doc-parser/src/PhpDocInfo/PhpDocInfo.php

```diff
@@ -282,20 +282,20 @@ public function getByType(string $type): ?PhpDocNode
     {
         $this->ensureTypeIsTagValueNode($type, __METHOD__);
 
-        foreach ($this->phpDocNode->children as $phpDocChildNode) {
-            if (is_a($phpDocChildNode, $type, true)) {
-                return $phpDocChildNode;
+        foreach ($this->phpDocNode->children as $child) {
```

Comment:
This information should be kept. It's a huge tell about the type compared to just `child`

Reply:
Fixed by skip property fetch not from `ThisType`